src/ITensors.jl

```diff
@@ -132,11 +132,18 @@ include("indexset.jl")
 # ITensor
 #
 include("itensor.jl")
+include("OPS/tensor_algebra.jl")
 include("broadcast.jl")
 include("decomp.jl")
 include("iterativesolvers.jl")
 include("adapt.jl")
 
+#####################################
+# Experimental ITensor Functions
+#
+include("experimental/itensorcombiner.jl")
```

Comment:
The `combiner` constructor isn't an experimental feature so this isn't a good location for this file.

Reply:
Okay, I though you had said this is a function which you were using to test a theory? I left the NDTensor based combine functions untouched in 'itensor.jl'

Reply:
This file was moved to tensor_operations

Reply:
The experimental feature I was referring to is the `combine_contract` function.